crates/ironrdp-client/src/rdp.rs

```diff
@@ -280,6 +280,7 @@ async fn active_session(
                 ActiveStageOutput::PointerBitmap(_) => {
                     // Not applicable, because we use the software cursor rendering.
                 }
+                ActiveStageOutput::DeactivateAll(_) => todo!("DeactivateAll"),
```

Comment:
todo: Port the logic from ironrdp-web

Reply:
https://github.com/Devolutions/IronRDP/pull/421/commits/2359d0030d88755c996f503538860011067e6ff5